### DISCUSSION TITLE
Language not showing up in repo - F#

### DISCUSSION THREAD

**Discussion by 64J0** [github-linguist/linguist#6407] [Classification] [answered]
## Describe the bug

I created this repository for a POC: https://github.com/64J0/custom-fsharp-gh-action-and-dynamic-matrix.

And it's not showing correctly the languages I have there.

![image](https://user-images.githubusercontent.com/50725287/236889931-e9c4170f-91f9-4a49-88bf-b7344cd8db7b.png)

The HCL it is recognizing correctly, but it's ignoring what I have inside the `.github/fsharp-custom-action/` folder. I have some programs written F# there (`src/` and `tests/`).

### Expected behaviour

Show that I have F# code in this repository along with HCL.

### Related discussion

* https://github.com/orgs/community/discussions/54854

**Comment by 64J0** [comment, marked as answer]
Problem solved by adding this [.gitattributes](https://github.com/64J0/custom-fsharp-gh-action-and-dynamic-matrix/blob/master/.gitattributes) file.

**Comment by lildude** [comment]
For this discovering this later, this is because the `.github` directory is considered [vendored](https://github.com/github-linguist/linguist/blob/e1c3110a4429d79deba370ceefda372dad03b003/lib/linguist/vendor.yml#L387) as it doesn't normally contain code
